apps/trading/components/navbar/navbar.tsx

```diff
@@ -59,7 +52,7 @@ interface NavLinkProps {
 
 const NavLink = ({ name, path, testId = name }: NavLinkProps) => {
   const router = useRouter();
-  const isActive = router.asPath === path;
+  const isActive = router.asPath.includes(path);
```

Comment:
sth weird happening here. it works for the trading tab but i should not be doing this. fails on the portfolio tab after refresh.